lib/src/main/scala/org/typelevel/scalacoptions/ScalacOptions.scala

```diff
@@ -157,6 +166,10 @@ private[scalacoptions] trait ScalacOptions {
   val languageStrictEquality =
     languageFeatureOption("strictEquality", version => version >= V3_0_0)
 
+  /** Enable dynamics support. */
```

Comment:
I'm also wondering is this worth adding, `dynamics` seems not a popular choice and it's not too much hassle to use `languageFeatureOption("dynamics")` directly.

Reply:
I'm using it in some project and I though i'd be useful - If isn't we can remove it, np